AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TestHarness/AmplifyAuthCognitoPluginTests.swift

```diff
@@ -14,7 +14,8 @@ class AmplifyAuthCognitoPluginTests: XCTestCase {
 
     let apiTimeout = 1.0
 
-    func testAuthCognitoPlugin() {
+    @MainActor
+    func testAuthCognitoPlugin() async {
```

Comment:
`async` bubbled up to this test from a change in the SDK. It needs to be `@MainActor` because `async` and `XCTContext.runActivity` do not play nicely together otherwise.